pkg/provider-local/local/types.go

```diff
@@ -26,6 +26,8 @@ const (
 	Name = "provider-local"
 	// Type is the type of resources managed by the local actuators.
 	Type = "local"
+	// BackupProviderType is the provider type that is used for backups.
+	BackupProviderType = "Local"
```

Comment:
```suggestion
	BackupProviderType = "local"
```

Reply:
As I said above the etcd-backup-restore is checking for `Local` with capital.